core/tracing.ts

```diff
@@ -0,0 +1,76 @@
+require("source-map-support").install(); // tslint:disable-line
```

Comment:
Here is the hack to get the tslint error, any suggestions on how to use "import"?